src/ucp/wireup/wireup.c

```diff
@@ -221,6 +221,12 @@ ucp_wireup_msg_send(ucp_ep_h ep, uint8_t type, const ucp_tl_bitmap_t *tl_bitmap,
 
     ucs_assert(ep->cfg_index != UCP_WORKER_CFG_INDEX_NULL);
 
+    if (ep->flags & UCP_EP_FLAG_FAILED) {
+        ucs_debug("ep %p: not sending WIREUP message (%u), because ep failed",
+                  ep, type);
+        return UCS_ERR_CONNECTION_RESET;
```

Comment:
wouldn't it cause other things to fail?

Reply:
some callers just ignore the return value, some of them (send REQUEST) handles and destroy UCP EP